### DISCUSSION TITLE
Discussion venues for Armada

### DISCUSSION THREAD

**Discussion by jayofdoom** [armadaproject/armada#1044] [Ideas]
Continuation from https://github.com/G-Research/armada/issues/1024

As Armada will be growing in contributor and user base, we should consider how to enable good communications between G-Research based contributors and contributors outside of the organization.

I did some research about how CNCF projects communicate, as a potential template for us to follow as a project that has applied for incubation(1). Of the 33 projects listed as incubating on the CNCF website; all but one have an easily found chat method.

The vast majority of those use slack for communication, with others using IRC, Discord, and Glitter. Some projects have their own Slack servers, but most use cloud-native.slack.com or slack.cncf.io.

Given, to my understanding, most developers currently working on this project are active in some Slack servers, using Slack seems like the logical solution.

I propose we agree to use a public Slack environment to communicate about Armada technical decisions and to allow a place for potential new users and contributors to collaborate with us. If we come to this agreement, I can reach out to the teams that run cloud-native.slack.com and/or slack.cncf.io if they'd be willing to host a channel about Armada, in order to make it simple for us to create the channel without having to lure folks to a new Slack server.

What do you all think?

1: https://docs.google.com/spreadsheets/d/1gDG8AKDouxDu94sgQccR6zhIkezO11eQXlhrg7XhG8w

**Comment by jayofdoom** [comment]
Based on responses sent to me privately in slack, and the comments on the original issue, I intend to activate on putting a Slack channel in CNCF slack -- if they will have us before we're technically incubating.

**Reply by stackedsax** [reply]
Thanks for the research -- that's really useful information regardless of this particular conversation.

And yes, just to pile on from my point of view: a Slack channel in the CNCF Slack workspace is kind of exactly why we applied for Sandbox status.  Based on your research, it looks like a third of those projects are either in the CNCF or k8s workspaces, and I think that's a strong signal.  Ok, there might be another 500 projects in the CNCF landscape that you could canvas, but the whole idea of Sandboxing is to give the wider k8s community visibility into projects like ours.  Maybe in time we'll move to our own Armada-specific workspace when the community grows so large, but Armada's particular community is so specific we will likely have a small community for a while.  Getting the occasional set of eyeballs from random CNCF members is definitely worth it.
